developer/requirements.md

```diff
@@ -2,22 +2,22 @@
 
 ## macOS
 
-- We suggest installation of [Xcode](https://developer.apple.com/xcode/downloads/)
+- To install the required build tools, Mac users must install [Xcode](https://developer.apple.com/xcode/downloads/)
```

Comment:
I don't think xcode is actually required. is it? I know that it used to prevent the `Failed to load c++ Json message`,  but not sure it's actually required.